docs/core/tools/dotnet-restore.md

```diff
@@ -49,6 +49,18 @@ For project-specific tooling, `dotnet restore` first restores the package in whi
 
 The behavior of the `dotnet restore` command is affected by some of the settings in the *Nuget.Config* file, if present. For example, setting the `globalPackagesFolder` in *NuGet.Config* places the restored NuGet packages in the specified folder. This is an alternative to specifying the `--packages` option on the `dotnet restore` command. For more information, see the [NuGet.Config reference](/nuget/schema/nuget-config-file).
 
+## Implicit `dotnet restore`
+
+Starting with .NET Core 2.x, `dotnet restore` is run implicitly if necessary when you issue the following commands:
```

Comment:
should we say 2.0 here since we know when it started?

I liked the text we worked for the announcement a little better: https://github.com/dotnet/announcements/issues/23